src/components/BottomPlayBar.tsx

```diff
@@ -0,0 +1,81 @@
+import { Image, Pressable, StyleSheet, Text, View } from 'react-native';
+import Icon from '../../assets/icons';
+
+const styles = StyleSheet.create({
+  card: {
+    backgroundColor: '#D9D9D9',
+    width: 400,
+    height: 78,
+    borderRadius: 5,
+    margin: 0,
+  },
+  title: {
+    fontSize: 12,
+    marginLeft: 15,
+    marginTop: 9,
+  },
+  meta: {
+    marginTop: 10,
+    marginLeft: 15,
+    fontSize: 9,
+    flexWrap: 'wrap',
+  },
+});
+
+type BottomPlayBarProps = {
+  onPress?: () => void;
+  children: React.ReactNode;
+  style?: object;
+};
+
+function BottomPlayBar({ onPress, children, style }: BottomPlayBarProps) {
+  return (
+    <Pressable style={[styles.card, style]} onPress={onPress}>
+      {children}
+    </Pressable>
+  );
+}
+
+type SearchBottomPlayBarProps = {
```

Comment:
I would rename this to `NowPlayingBar` or something for consistency with your wrapper naming. Also bc this bar doesn't actually handle any of the positioning logic, so including the word 'bottom' could be misleading.